csharp/ql/src/semmlecode.csharp.dbscheme

```diff
@@ -1699,3 +1699,9 @@ cil_attribute_positional_argument(
 @dotnet_int_literal = @integer_literal_expr | @cil_ldc_i;
 @dotnet_float_literal = @float_literal_expr | @cil_ldc_r;
 @dotnet_null_literal = @null_literal_expr | @cil_ldnull;
+
+@metadata_entity = @cil_method | @cil_type | @cil_field | @cil_property | @field | @property |
+    @callable | @value_or_ref_type | @void_type;
+
+#keyset[entity, location]
+metadata_handle(int entity : @metadata_entity ref, int location: @assembly ref, int handle: int ref)
```

Comment:
Is `#keyset[location, handle]` also valid?

Reply:
No, because there are 1-2 entities for each (location,handle) - the CIL one and the C# one.

Reply:
Makes sense.